libjpeg-turbo.lua

```diff
@@ -0,0 +1,606 @@
+    -- configuration { "linux" }
+    -- -------------------------------------------------------------
+
+    -- common configuration settings
+
+    dofile (_BUILD_DIR .. "/static_linux.lua")
+
+    -- project specific configuration settings
+
+    configuration { "linux" }
+
+      files {
+        "jsimd_none.c",
+      }
+
+    -- -------------------------------------------------------------
+    -- configuration { "linux", "Debug", "x64" }
+    -- -------------------------------------------------------------
+
+    -- common configuration settings
+
+    dofile (_BUILD_DIR .. "/static_linux_x64_debug.lua")
+
+    -- project specific configuration settings
+
+    -- configuration { "linux", "Debug", "x64" }
+
+    -- -------------------------------------------------------------
+    -- configuration { "linux", "Release", "x64" }
+    -- -------------------------------------------------------------
+
+    -- common configuration settings
+
+    dofile (_BUILD_DIR .. "/static_linux_x64_release.lua")
+
+    -- project specific configuration settings
+
+    -- configuration { "linux", "Release", "x64" }
+
+    -- -------------------------------------------------------------
+  end
+
+  if (os.is("macosx") and not _OS_IS_IOS and not _OS_IS_ANDROID) then
+    -- -------------------------------------------------------------
+    -- configuration { "macosx" }
+    -- -------------------------------------------------------------
+
+    -- common configuration settings
+
+    dofile (_BUILD_DIR .. "/static_mac.lua")
+
+    -- project specific configuration settings
+
+    configuration { "macosx" }
+
+      files {
+        "jsimd_none.c",
+      }
+
+    -- -------------------------------------------------------------
+    -- configuration { "macosx", "Debug", "x64" }
+    -- -------------------------------------------------------------
+
+    -- common configuration settings
+
+    dofile (_BUILD_DIR .. "/static_mac_x64_debug.lua")
+
+    -- project specific configuration settings
+
+    -- configuration { "macosx", "Debug", "x64" }
+
+    -- -------------------------------------------------------------
+    -- configuration { "macosx", "Release", "x64" }
+    -- -------------------------------------------------------------
+
+    -- common configuration settings
+
+    dofile (_BUILD_DIR .. "/static_mac_x64_release.lua")
+
+    -- project specific configuration settings
+
+    -- configuration { "macosx", "Release", "x64" }
+
+    -- -------------------------------------------------------------
+  end
+
+  if (_OS_IS_IOS) then
+    -- -------------------------------------------------------------
+    -- configuration { "ios" } == _OS_IS_IOS
+    -- -------------------------------------------------------------
+
+    -- common configuration settings
+
+    dofile (_BUILD_DIR .. "/static_ios.lua")
+
+    -- project specific configuration settings
+
+    configuration { "ios*" }
+
+      defines {
+        "_REENTRANT",
```

Comment:
Are these defines needed?

Reply:
Looks like these (and the android defines) were brought over from the old `jpeg.lua`...I'll remove them and confirm that everything still builds.